django_feature_flip/tests.py

```diff
@@ -49,7 +49,9 @@ def test_enabled_for_non_existent_feature(self):
         try:
             self.feature_flip.enabled('non_existent_feature', actor)
         except FeatureFlip.FeatureFlipError as error:
-            self.assertEqual(error.message, 'Feature non_existent_feature does not exist')
+            self.assertEqual(
+                error.message, 'Feature non_existent_feature does not exist'
```

Comment:
I think this should have a trailing comma, but it looks like black removes it. There is an issue about it: https://github.com/psf/black/issues/274. Is there a config that can be added so this doesn't happen?

Reply:
Mmm, not quite sure it has a bug. According to PEP8: [Trailing commas are usually optional, except they are mandatory when making a tuple of one element ](https://www.python.org/dev/peps/pep-0008/#id29)

Reply:
Yeah I know that they are optional but we were choosing to add them in these type of cases. But we can adapt to what black does. We just need to make sure that black and flake8 don't contradict each other. 👍

Reply:
The idea is to keep our standards, not to adapt to black. I think in this particular case, the trailing comma does not apply because there are several arguments within one line. I'll change it to one element per line ✅